com.sap.cloud.lm.sl.cf.api/src/main/java/com/sap/cloud/lm/sl/cf/web/api/model/MessageType.java

```diff
@@ -1,39 +1,18 @@
 package com.sap.cloud.lm.sl.cf.web.api.model;
 
 import javax.xml.bind.annotation.XmlEnum;
-import javax.xml.bind.annotation.XmlEnumValue;
 import javax.xml.bind.annotation.XmlType;
 
 @XmlType(name = "")
 @XmlEnum(MessageType.class)
 public enum MessageType {
 
-    @XmlEnumValue("INFO")
-    INFO(String.valueOf("INFO")), @XmlEnumValue("ERROR")
```

Comment:
This is generated code. Avoid touching it. Furthermore, are you sure this will produce the same XML as before?

Reply:
Yes, the docs for xml enum and enum value state that "if no xmlEnumValue annotation is present on the enum values in a class annotated with xmlEnum, the Enum.name() method will be used for the xml mapping".
And I see here that we are using the same strings as the names of the enum values. That is why I removed it